contracts/Bitmask.sol

```diff
@@ -0,0 +1,21 @@
+pragma solidity ^0.4.22;
+
+library Bitmask {
```

Comment:
Should probably add some tests of this contract.

Reply:
See separate PR